tala_connector.py

```diff
@@ -753,15 +771,15 @@ def finalize(self):
     username = args.username
     password = args.password
 
```

Comment:
Add a new command line argument as `verify`. Refer IP Control app for the same.

Reply:
Added command-line argument as verify.